langserver/format.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	"github.com/pmezard/go-difflib/difflib"
 	"golang.org/x/tools/go/buildutil"
+	"golang.org/x/tools/imports"
```

Comment:
i don't think this is in our vendor tree. Can you run `dep ensure`?

Reply:
It is: https://github.com/sourcegraph/go-langserver/tree/master/vendor/golang.org/x/tools/imports